packages/perseus-editor/src/editor-page.tsx

```diff
@@ -156,7 +156,7 @@ class EditorPage extends React.Component<Props, State> {
 
         this.itemEditor.current?.triggerPreviewUpdate({
             type: "question",
-            data: _({
```

Comment:
I hadn't seen this usage of `_` (underscore) before. It provides an "OO" wrapper for the object.